app/assets/javascripts/models/data/dataset.js.coffee

```diff
@@ -97,7 +97,13 @@ ns.Dataset = do (ko
 
     thumbnail: ->
       granule = @browseable_granule
-      "#{scalerUrl}/#{granule}?h=85&w=85" if granule?
+      collection_id = @id for link in @links when link['rel'].indexOf('browse#') > -1
+      if granule?
```

Comment:
Can we flip this logic around and make the collection thumbnail take precedence?